bin/build

```diff
@@ -29,4 +29,9 @@ docker build --build-arg VERSION="$VERSION" \
              -t "${TAG}-redhat" \
              .
 
+docker build --build-arg VERSION="$VERSION" \
+             -f Dockerfile.fips \
+             -t "${TAG}-fips" \
+             .
```

Comment:
this isn't going to result in the outcome you're looking for

we should probably revise the pipeline here so that we have a single Dockerfile for the current authenticator image and the RH image (see [the secretless Dockerfile here](https://github.com/cyberark/secretless-broker/blob/master/Dockerfile))

then the FIPS image Dockerfile can be a version of that where the `authenticator-builder` base image is `goboring/golang:1.12.17b4` (and the 3rd RH image reference is dropped, unless we also want a FIPS RH image)

what it's doing right now is:
- building the binary builder image based on Dockerfile
- running the binary builder image to build the binary and output it locally
- building the dockerhub image from Dockerfile.scratch (which copies the binary in)
- building the redhat image from Dockerfile.redhat (which copies the binary in)
- building the FIPS image (which has no binary once it's built, since it is not copied - it would have to be more like Dockerfile.scratch for that to work)

I'd love to see this become two dockerfiles, Dockerfile and Dockerfile.fips, that include multi-stage builds - eg each defines a builder image based on the proper golang version, and then builds the final dockerhub and RH images

lmk if you have questions about what I mean by this